src/annotation.ts

```diff
@@ -11,28 +11,50 @@ import { Vex } from './vex';
 import { Flow } from './tables';
 import { Modifier } from './modifier';
 import { TextFont } from './textfont';
+import { FontInfo } from './types/common';
+import { StemmableNote } from './stemmablenote';
+import { ModifierContextState } from './modifiercontext';
 
 // To enable logging for this class. Set `Vex.Flow.Annotation.DEBUG` to `true`.
-function L(...args) {
+function L(
+  // eslint-disable-next-line
+  ...args: any []) {
   if (Annotation.DEBUG) Vex.L('Vex.Flow.Annotation', args);
 }
 
+export enum Justify {
+  LEFT = 1,
+  CENTER = 2,
+  RIGHT = 3,
+  CENTER_STEM = 4,
+}
+
+export enum VerticalJustify {
+  TOP = 1,
+  CENTER = 2,
+  BOTTOM = 3,
+  CENTER_STEM = 4,
+}
 export class Annotation extends Modifier {
-  static get CATEGORY() {
+  static DEBUG: boolean;
+
+  note?: StemmableNote;
+
+  protected justification: Justify;
+  protected vert_justification: VerticalJustify;
+  protected text: string;
+  protected font: FontInfo;
+
+  static get CATEGORY(): string {
```

Comment:
We should fix these everywhere, since TS supports static properties: `static CATEGORY = 'annotations'`

Reply:
This would have to be done in another PR changing all of them. If I only change here I get:

```
Uncaught TypeError: Cannot set property CATEGORY of function Modifier() {
        var _this = _super.call(this) || this;
        _this.setAttribute('type', 'Modifi...<omitted>... } which has only a getter
```